test/browser/component.js

```diff
@@ -660,3 +660,27 @@ describe(`Rendering exception`, function() {
     expect(el.textContent).to.contain(`Value of foo.bar: later success`);
   });
 });
+
+context.only(`$hooks`, function() {
+  context(`.delayedAttrRemove`, function() {
+    it(`sets attr open:false immediately and removes modal after delay`, async function() {
```

Comment:
```suggestion
    it(`sets attr immediately and removes element after delay`, async function() {
```
(docstring should describe the behavior of the system under test, not the specifics of this case's fixtures)